nuxeo-tools-hooks/nxtools/hooks/services/github_service.py

```diff
@@ -349,7 +349,10 @@ def github_notify(self, pull_request):
             for key in keys:
                 parts.append("- %s" % self._get_issue_comment(key, jira))
         if parts:
-            return pull_request.gh_object.create_issue_comment("\n".join(parts))
+            try:
+                return pull_request.gh_object.create_issue_comment("\n".join(parts))
+            except Exception as e:
+                log.error('github_notify: Failed to comment PR %s: %s', pull_request.gh_object.html_url, e)
```

Comment:
```suggestion
            except Exception:
                log.exception('github_notify: Failed to comment PR %s', pull_request.gh_object.html_url)
```
Note: `log.exception(...)` is just the same as `log.error(..., exc_info=True)`.